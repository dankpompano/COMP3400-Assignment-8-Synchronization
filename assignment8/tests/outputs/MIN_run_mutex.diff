--- outputs/MIN_run_mutex.txt	2025-04-21 11:47:15.971019870 -0400
+++ expected/MIN_run_mutex.txt	2020-04-17 17:22:24.000000000 -0400
@@ -0,0 +1,2 @@
+Running mutual exclusion test
+Value of shared: 0
