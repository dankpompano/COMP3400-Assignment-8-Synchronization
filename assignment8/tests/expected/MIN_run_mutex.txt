Running mutual exclusion test
Value of shared: 0
